docs/en/docs/sql-manual/sql-functions/aggregate-functions/collect_list.md

```diff
@@ -30,9 +30,8 @@ under the License.
 
 `ARRAY<T> collect_list(expr)`
 
-Returns an array consisting of all values in expr within the group.
-The order of elements in the array is non-deterministic. NULL values are excluded.
-
+Returns an array consisting of all values in expr within the group, and ,with the optional `max_size` parameter limits the size of the resulting array to `max_size` elements.The order of elements in the array is non-deterministic. NULL values are excluded.
+It has an alias `group_array`.
```

Comment:
`The order of elements in the array is non-deterministic`, so the groovy test case of result  is also unstable, maybe be failed in other PR

Reply:
Non-deterministic in docs refers to the result order compared to which in the passed in columns, however, from the collect function's perspective, it is idempotent, and will not result in test failure.

Reply:
ennnn... here I'm not sure is that, if have 100 rows in column, but when set max_size = 1, whether result will be same always

Reply:
According to several regression tests up to now, the problem didn't come up.

Reply:
if you have more replica in different BE，the result will be not same. so you must add stable case in regression test in any env. just `length（arry）` to make sure res is stable

Reply:
I see, I'll modify it.